docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,63 @@
 [[run-digger-installer]]
-== Run Digger Installer
+=== Run Digger Installer
 
-.To run the Digger installer, follow these steps:
+. Clone https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^].
 
-. Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
+. Make a copy of the Ansible https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[inventory-sample^], and update it with information about your infrastructure.
 
-. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
+NOTE: For a full list of variables that can be configured via the inventory file, refer to <<installer-variables>>.
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
+. Use one of these commands to run the playbook and follow the instructions printed in the terminal:
+//+
+//Refer to https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^] to see the roles executed during installation and their associated tags.
 
-** Command to install digger without a macOS server
+** Command to install Digger with macOS server
++
+WARNING: Make sure your macOS Server is prepared for installation. See link:macos-prereqs[macOS Prerequisities].
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
-
-** Command to install digger with a macOS server
++
+** Command to install Digger without a macOS server
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx
 ----
+WARNING: As of Jan 4th 2018, only `provision-osx` tag is working with with `--skip-tags`.
 
-. Carry out manual steps during the installation
 +
-There are three points during the installation where there are manual steps required.
+//** Roles can be run individually or in batch with flag //--tags=<role_tag_1>,...,<role_tag_x>
+//+
+//----
```

Comment:
can this section be removed as it's commented out?

Reply:
no. I expect this to be supported in the future.

resolved